101-vpn-custom-ipsec-policy/azuredeploy.json

```diff
@@ -0,0 +1,203 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location" : {
```

Comment:
add a metadata description to the location param

Reply:
addressed in commit 604d1ab